aunt-leahs-app/src/pages/homepage/homepage.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Head from '../../components/header/header';
+import LocationSelection from '../../components/locationTable/locationTable';
+import CustomButton from '../../components/customButton/customButton'
```

Comment:
The pathing for the three components above seems unnecessarily long. For example, why not '../../components/customButton' ?